utils/flare.py

```diff
@@ -257,16 +295,26 @@ def _strip_password(self, file_path):
         return temp_path, password_found
 
     # Add output of the command to the tarfile
-    def _add_command_output_tar(self, name, command):
-        temp_path = os.path.join(tempfile.gettempdir(), name)
+    def _add_command_output_tar(self, name, command, logger_name=None):
+        temp_path = os.path.join(tempfile.gettempdir(), name.replace(os.sep, '__'))
```

Comment:
I'm not a huge fan of this (especially just after a `os.path.join`, it's kind of redondant), maybe instead of using the `tempfile.gettempdir()` you could call https://docs.python.org/2/library/tempfile.html#tempfile.mkstemp ? Or add a `subdir=` option. Or even easier a `temp_name=` option.